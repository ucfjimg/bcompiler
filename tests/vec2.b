main()
{
    extrn u, v, w, printf;

    w[0][4] = 42;
    w[1][6] = 0100; /* @ sign */

    printf("The answer is dadams%d%clocalhost*n", u[4], v[6]);

}

u[10];
v[10];
w[] u, v;
