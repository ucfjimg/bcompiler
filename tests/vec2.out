The answer is dadams42@localhost
